x-pack/plugins/fleet/common/openapi/bundled.json

```
@@ -4191,6 +4201,28 @@
             "$ref": "#/components/responses/error"
           }
         },
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string",
+                    "description": "The name of the enrollment API key. Must be unique."
+                  },
+                  "policy_id": {
+                    "type": "string",
+                    "description": "The id of the agent policy the Elastic Agent will be enrolled in."
```

```suggestion
                    "description": "The ID of the agent policy the Elastic Agent will be enrolled in."
```
Just a small thing, but I think this and the other instances of `id` would be clearer as `ID`.